docs/source/release/v6.13.0/Diffraction/Engineering/New_features/39139.rst

```diff
@@ -0,0 +1 @@
+- `#38882 <https://github.com/mantidproject/mantid/issues/39139>`_ : Refactor how ``GSAS-II`` files are found to avoid use of hard-coded paths which could cause issues going forward should other files neccessary for import of ``GSAS-II`` be relocated in the future like ``GSASIIscriptable.py`` was for versions 5758 and later.
```

Comment:
The link is to 39139 but the text is 38882. Is that deliberate?

Reply:
No, this is just copied over as a placeholder while the PR is still a draft. I will be updating this before stating it is ready for review.

Reply:
I'm just wondering if this is what is causing the docs build to fail.

Reply:
Just pushed up an updated version so we can check. If it passes, I would be interested to know exactly how the docs failed. If it's because of a duplicate issue tag, I wonder if this is desirable functionality or not for the rare edge case where multiple PRs link to the same issue. This would could be a possibility if the PRs made sense to be broken into sections and have separate release notes to avoid having one really large PR and equally large release note to describe changes if all the work is related to one issue. - though there is a good argument that if this ever occurred, that separate sub-issues should be created.